src/podcast_processor/transcribe.py

```diff
@@ -145,18 +167,29 @@ def split_file(
             chunk.export(export_path, format="mp3")
             chunks.append((export_path, start_offset_ms))
 
+        self.logger.info(f"Split into {len(chunks)} chunks")
+
         return chunks
 
     def get_segments_for_chunk(self, chunk_path: str) -> List[TranscriptionSegment]:
         with open(chunk_path, "rb") as f:
+
+            self.logger.info(f"Transcribing chunk {chunk_path}")
+
             transcription = self.openai_client.audio.transcriptions.create(
-                model="whisper-1",
+                model=self.config.remote_whisper_model,
                 file=f,
-                timestamp_granularities=["segment"],
-                language="en",
+                # Faster-whisper-server uses segments instead of segment
+                timestamp_granularities=["segments" if self.config.faster_whisper_server else "segment"],
```

Comment:
https://github.com/fedirz/faster-whisper-server/issues/19 seems like this may be fixed?

Reply:
it really depends on the fork. But, I think your right, it's probably safe to drop this. I fixed that in my fork anyway.